dali/pipeline/operators/detection/box_encoder.cc

```diff
@@ -165,6 +201,20 @@ encoded box.")code")
     .AddOptionalArg(
         "criteria",
         R"code(Threshold IOU for matching bounding boxes with anchors. Value between 0 and 1.)code",
-        0.5f, false);
+        0.5f, false)
+    .AddOptionalArg(
+        "offset",
+        R"code(Returns offset in `EncodedBBoxes` (encoded_bboxes - anchors).
+               Optional normalization by providing `std`, `mean` and `scale`.)code",
```

Comment:
```suggestion
    R"code(Returns offset in `EncodedBBoxes` (encoded_bboxes - anchors)and perform
        normalization by providing `std`, `mean` and `scale` (the default values are transparent).)code",
```

Reply:
I hope this explains better what is happening:
```
Returns normalized offsets `(encoded_bboxes*scale - anchors*scale) - mean) / stds`
               in `EncodedBBoxes` using `std`, `mean` and `scale` arguments (default values are transparent)
```